lua/core/default_config.lua

```diff
@@ -160,7 +164,7 @@ M.mappings.plugin = {
       new_file = "<leader>fn", -- basically create a new buffer
       open = "<leader>db", -- open dashboard
       session_load = "<leader>l", -- load a saved session
-      session_save = "<leader>s",
+      session_save = "<leader>s", -- save a session
```

Comment:
no need to add a comment there , also remove the above comment too .

session_save/ load itself makes sense .

Reply:
Hmmmmm, I'm okay with being a little verbose with comments & explanations here, but if you really think I'll happily remove it!

Reply:
Being a lil verbose doesn't harm at all

Reply:
> Hmmmmm, I'm okay with being a little verbose with comments & explanations here, but if you really think I'll happily remove it!

yes remove the comments , the var names are already very self-explanatory

Reply:
M. But it doesn't harm adding more verbose comments. It is used by the user right so .

Reply:
It's `default_config` so we can always be verbose, but I'll leave this open for now as I should re-do all the comments in that file anyway